frontend/src/Login.jsx

```diff
@@ -1,29 +1,39 @@
+import { useNavigate } from "react-router-dom";
+import InputBox from "./components/InputBox";
+
 const Login = () => {
+  const navigate = useNavigate();
+
   return (
     <div className="flex flex-col">
       <header>
-        <div className="text-center text-5xl m-30 font-bold text-indigo-50 text-shadow-lg">
+        <div className="text-center text-8xl m-20 font-bold text-indigo-50 text-shadow-lg">
```

Comment:
What's up with these class names? Is this a product of using some CSS framework? 

My worry is that these will rapidly become unmaintainable because it's so hard to figure out what these are supposed to mean at a glance. It's possible this is normal and I just don't know enough frontend to say.